docs/reference/tmx-map-format.md

```diff
@@ -19,6 +19,7 @@ Have a look at the [changelog](tmx-changelog.md) when you're interested in what
 * <b>height:</b> The map height in tiles.
 * <b>tilewidth:</b> The width of a tile.
 * <b>tileheight:</b> The height of a tile.
+* <b>hexsidelength:</b> For staggered and hexagonal maps, determines the amount of width or height (depending on the staggered axis) of the tile's edge, in pixels.
```

Comment:
Thanks for adding this! But I think the "staggered and" should be removed here. I'm just wondering why you wrote that there? In my testing and as is intended, only hexagonal maps use this property.